pymc/logprob/transforms.py

```diff
@@ -391,7 +419,7 @@ def measurable_transform_logprob(op: MeasurableTransform, values, *inputs, **kwa
         jacobian = jacobian.sum(axis=tuple(range(-ndim_supp, 0)))
 
     # The jacobian is used to ensure a value in the supported domain was provided
-    return pt.switch(pt.isnan(jacobian), -np.inf, input_logprob + jacobian)
+    return pt.switch(pt.isnan(input_logprob + jacobian), -np.inf, input_logprob + jacobian)
```

Comment:
Can we revert this change?

Reply:
Potentially let me check. The reason I did that was because it meant that we return `-np.inf` consistently when `input_logprob = nan`, which is the case for some of the transforms.

Reply:
Okay seems that isn't the case anymore and tests pass with the reverted change